src/_P025_ADS1115.ino

```diff
@@ -170,6 +170,8 @@ boolean Plugin_025(uint8_t function, struct EventStruct *event, String& string)
         if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
           log  = F("ADS1115 : Analog value: ");
           log += value;
+          log += F(" / Channel: ");
+          log += P025_data->getMux();
```

Comment:
Instead of adding an entire access method to fetch the multiplexer config from the `P025_plugin_struct`, you could have simply appended `PCONFIG(2)` to the log...